AmazonKinesisVideoDemoApp/build.gradle

```diff
@@ -49,7 +49,7 @@ repositories {
 
 
 dependencies {
-    def aws_version = '2.30.+'
+    def aws_version = '2.36.+'
```

Comment:
do you want to update this to latest?

Reply:
From my testing, `2.36.+` and `2.52.+` (latest) work the same (as expected, as nether of those 4 dependencies have been updated since `2.31.+`).

Feel free to update to the latest if you wish.

Reply:
Thanks for the testing! Updated to 2.52.+ as proposed.